backend/tasks/tasks.controller.js

```diff
@@ -18,33 +18,59 @@ async function readById(req, res) {
 }
 
 async function create(req, res) {
-  const newItem = req.body;
+  /*const newItem = req.body;
 
   if (!newItem || !newItem.title) {
     return res.status(400).send("Não se esqueça do `title`!");
+  }*/
+  const { error, value } = createTaskSchema.validate(req.body);
+
+  if (error) {
+    return res.status(400).send(error.details.map(detail => detail.message).join(", "));
   }
 
-  await service.create(newItem);
-  res.status(201).send(newItem);
+  await service.create(value);
+  res.status(201).send(value);
+
+  /*await service.create(newItem);
+  res.status(201).send(newItem);*/
 }
 
 async function updateById(req, res) {
   const id = req.params.id;
-  const newItem = req.body;
+  const { error, value } = updateTaskSchema.validate(req.body);
+
+  if(error){
```

Comment:
Acredito que valha adicionar um formatador automático como o Prettier.
```suggestion
  if (error){
```